pkg/reconciler/v1alpha1/route/table_test.go

```diff
@@ -1503,11 +1503,12 @@ func setDomain(route *v1alpha1.Route, domain string) *v1alpha1.Route {
 	return route
 }
 
-func simpleNotReadyConfig(namespace, name string) *v1alpha1.Configuration {
+func simpleNotReadyConfig(namespace, name, version string) *v1alpha1.Configuration {
```

Comment:
Do you mind just hard coding the version in this function that builds the configuration fixture? Then the diff would be smaller?

Reply:
You mean hardcode the version here but having a version parameter to generate wanted patch?

Reply:
Yup

Reply:
Done. But I really don't understand the point of specifying the version in these functions but not the others.

Reply:
> But I really don't understand the point of specifying the version in these functions but not the others.

Ideally I would want the setup of the relevant tests to specify the `resourceVersion` explicitly. 
Modifying `simpleNotReadyConfig` does this but it's misleading in other tests since you might think that property has some special meaning.

In hindsight it might have made sense to accomplish this with another helper method and use it in the relevant tests 

ie.
```
func setResourceVersion(config *Configuration, version string) *Configuration {
  config.ObjectMeta.ResourceVersion = version
}
```